tests/TargetList/test_stellar_mass.py

```diff
@@ -0,0 +1,309 @@
+import unittest
```

Comment:
tests are good, but please remove all `print()` statements - these are not needed inside unit tests.